src/common/MemberSelector/MemberSelector.tsx

```diff
@@ -87,7 +87,7 @@ const MemberSelector = ({ control, name }: SelectFieldProps) => {
           <FormControl>
             <Select
               {...field}
-              value={loginUser.role === UserRole.Member  ? loginUser.name : field.value}
+              value={loginUser.role === UserRole.Member  ? loginUser.id.toString() : field.value}
```

Comment:
This is bit weird. I talked to @ghulamhafeez that this is anti-pattern. Let's not make it part of this PR.